board/views.py

```diff
@@ -226,6 +228,38 @@ def delete_workspace(request):
     db.delete('workspace',id=ObjectId(wid))
     return JsonResponse("Deleted",safe=False)
 
+
+def _serve(filename,file_path):
+    """Serves a file"""
+    if not filename or not file_path:
+        raise InvalidArgumentException
+
+    response = HttpResponse(content_type='application/force-download') 
+    response['Content-Disposition'] = 'attachment; filename=%s' % smart_str(filename)
+    response['X-Sendfile'] = smart_str(file_path)
+    
+    return response
+
+
+def download_model(request):
+    """Serves a model file with model-id:mid"""
+    mid = request.GET.get('mid','')
+    if not mid:
+        return JsonResponse("",safe=False)
+
+    print("requested mdoel file for",mid)   
+    
+    filename = "model.ckpt.index"
+    file_path = "data/"+mid+"/{0}".format(filename)
```

Comment:
We have multiple ckpts for a said model. Does this function download the most recent model.ckpt?

Reply:
This PR does not address downloads. Issue is still open : https://github.com/mohit-chawla/recboard/issues/35